src/pages/gen2/build-a-backend/add-aws-services/analytics/index.mdx

```diff
@@ -0,0 +1,58 @@
+export const meta = {
+  title: 'Analytics',
+  description: 'Learn how to set Analytics resource powered by Pinpoint'
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+Amplify enables you to collect analytics data for your App. The Analytics category uses Amazon Cognito Identity pools to identify users in your App. Cognito allows you to receive data from authenticated, and unauthenticated users in your App. 
```

Comment:
the capitalization of "App" stands out. Should this be just "app" to align with the quickstart language? 
![image](https://github.com/aws-amplify/docs/assets/5033303/ca98e9eb-c1a0-4005-85a3-84731252f01d)